### PR TITLE
Adding new member Khanjan

### PR DESCRIPTION
**Pull request by Khanjan04**
adding info of new member-Khanjan

### CODE REVIEW

File: members/khanjan/data.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "last_name": "Varma",
+    "yoe": 0,
+    "company": "Vishwakarma Institute of Technology",
+    "designation": "student",
+    "img": "./img.png",
+    "github_id": "Khanjan04",
+    "linkedin_id": "khanjan-varma-5b3394183",
+    "twitter_id": "Khanjan54690482",
+    "instagram_id": "imkpv_",
+    "site": ""
```
Comment (swarajpure) [inline members/khanjan/data.json:13]: `site` can be removed if it doesn't exist.

**Review by swarajpure — CHANGES_REQUESTED** [review]
One change requested in data.json

Also, please read the rules for profile photo:
```
Images should be a square
Size of the image should be greater than 500px X 500px
The face should be visible, centered and identifiable
Face should cover at least 60% of the image
```

File: members/khanjan/data.json (hunk 1 above)
Comment (Khanjan04) [reply members/khanjan/data.json]: ok

**Review by swarajpure — APPROVED** [review]
Great! Thank you!
Congratulations on your first pull request in Real Dev Squad! 🎉

**Pull request merged**